app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!, only: [:create, :destroy]
+    
+  # POST "/microposts/20/comments"
+  def create
+    micropost = Micropost.find(params[:micropost_id])
```

Comment:
mostly we should use find_by to avoid unnecessary exceptions unless there is a strong need to do this